tera-templates/core/about.html

```diff
@@ -26,6 +26,10 @@ <h1 id="crate-title">About Docs.rs</h1>
             <a href="{{ docsrs_repo }}/issues">issues page</a>
             and file a new issue if you don't see an existing request.
             {%- endif -%}
+
+            Docs.rs builds crates with environment variable `DOCS_RS` set to
```

Comment:
```suggestion
            </p>
            <p>
            Docs.rs builds crates with environment variable `DOCS_RS` set to
```

Reply:
This is what it looks like currently:
![image](https://user-images.githubusercontent.com/23638587/85569931-f5e67d80-b600-11ea-9c77-9f4714625ee5.png)

Reply:
Sorry for that, nits applied.